static/css/impala/buttons.less

```diff
@@ -298,9 +298,44 @@ button[disabled],
     color: #919497;
     .box-shadow(0 3px rgba(0, 0, 0, 0.05), 0 -4px rgba(0, 0, 0, 0.05) inset);
     text-shadow: 0 1px 0 rgba(255,255,255,.5);
+    white-space: normal;
```

Comment:
I was unsure where the css was being compiled from. When changing the page which needs to be changed, the results aren't actively reflected in the local version because all css is compiled from other files using less. Any ideas of where to drop this line @magopian ?